### PR TITLE
Updated documentation for use alongside the WSUS admin-site

### PR DESCRIPTION
**Pull request by we-mi**
Added steps to make the Chocolatey Simple Server work alongside a WSUS admin-site (powershell and manual way)

Changed original references to WSUS for the new setup.

Updated ToC

### CODE REVIEW

File: How-To-Set-Up-Chocolatey-Server.md [hunk 1]
```diff
@@ -203,6 +204,46 @@ To configure for performance, you will want to do the following:
...
 
 
+### Configure Simple Server alongside WSUS admin-site
+
+If you are running the Simple Server next to a WSUS admin-site, you need to change some settings in the IIS-Configuration for making the Simple Server work correctly again.
+This is due to the fact that the WSUS admin-site installation registers two modules in the IIS root configuration, which will be passed down to all other sites running on that IIS including the Chocolatey Simple Server site.
+
+So for making the chocolatey simple server work again you need to disable inheritance for these modules in order to remove them from the chocolatey Simple Server site.  
+
+*Before making changes to your ISS configuration, read all steps carefully and make sure that you have a backup*
```
Comment (gep13) [inline How-To-Set-Up-Chocolatey-Server.md]:
```suggestion
*Before making changes to your IIS configuration, read all steps carefully and make sure that you have a backup*  
```

File: How-To-Set-Up-Chocolatey-Server.md [hunk 2]
```diff
@@ -203,6 +204,46 @@ To configure for performance, you will want to do the following:
 We are looking to add support for the package source to automatically handle this aspect - http://blog.nuget.org/20150922/Accelerate-Package-Source.html
 
 
+### Configure Simple Server alongside WSUS admin-site
+
+If you are running the Simple Server next to a WSUS admin-site, you need to change some settings in the IIS-Configuration for making the Simple Server work correctly again.
+This is due to the fact that the WSUS admin-site installation registers two modules in the IIS root configuration, which will be passed down to all other sites running on that IIS including the Chocolatey Simple Server site.
```
Comment (gep13) [inline How-To-Set-Up-Chocolatey-Server.md]:
```suggestion
This is due to the fact that the WSUS admin-site installation registers two modules in the IIS root configuration, which will be passed down to all other sites running on that IIS instance, including the Chocolatey Simple Server site.
```

File: How-To-Set-Up-Chocolatey-Server.md [hunk 3]
```diff
@@ -203,6 +204,46 @@ To configure for performance, you will want to do the following:
...
+
+So for making the chocolatey simple server work again you need to disable inheritance for these modules in order to remove them from the chocolatey Simple Server site.
```
Comment (gep13) [inline How-To-Set-Up-Chocolatey-Server.md]:
```suggestion
To make the Chocolatey Simple Server work again, you will need to disable inheritance for these modules in order to remove them from the Chocolatey Simple Server site.  
```

File: How-To-Set-Up-Chocolatey-Server.md [hunk 4]
```diff
@@ -203,6 +204,46 @@ To configure for performance, you will want to do the following:
...
+or manual:  
+  1. Open the IIS Management Console
+  2. Click on the server name on the upper left pane
+  3. On the right pane doubleclick on `Modules`. All available modules will show up
+  4. Search for `DynamicCompressionModule`, right-click it and choose `Unlock`
+  5. Do the same for `StaticCompressionModule`
+  6. Under `Sites` on the left pane click on the site for your Chocolatey Simple Server, most likely `ChocolateyServer`
+  7. Again, search for `Modules` on the right pane and doubleclick it
+  8. Search for `DynamicCompressionModule`, right-click it and choose `Remove`
+  9. Restart the Simple Server site
```
Comment (gep13) [inline How-To-Set-Up-Chocolatey-Server.md]:
```suggestion
  9. Restart the Chocolatey Simple Server site
```

File: How-To-Set-Up-Chocolatey-Server.md (hunk 1 above)
Comment (we-mi) [reply How-To-Set-Up-Chocolatey-Server.md]: OK...apparently Chocolatey has not made it into space (yet ;) )

**Comment by we-mi** [comment]
@gep13 Thank you for your appreciation :)

I committed the changes you suggested

**Comment by we-mi** [comment]
Hey @gep13,

is there anything else I can do to help with getting this pull request accepted?

Greetings,
Michael

**Comment by ferventcoder** [comment]
@we-mi thanks! Apologies this sat for a while. I've merged this in just now and it should be live on the site at some point soon.

**Pull request merged**